meta/saiserialize.cpp

```diff
@@ -951,6 +957,14 @@ std::string sai_serialize_queue_attr(
     return sai_serialize_enum(attr, &sai_metadata_enum_sai_queue_attr_t);
 }
 
+std::string sai_serialize_macsec_sa_attr(
+        _In_ const  sai_macsec_sa_attr_t attr)
```

Comment:
this is struct, use "&" in input parameter to not copy this

Reply:
But you can see above function `sai_serialize_queue_attr`, why does it need a special function for attribute enum type? I guess it looks like a convention.